src/Discovery/DiscoveryListView.tsx

```diff
@@ -62,9 +62,13 @@ export const DiscoveryListView: React.FunctionComponent<DiscoveryListViewProps>
     }
     // combine manifests from all selected studies
     const manifest = [];
-    props.selectedResources.forEach((study) => {
+    props.selectedResources.forEach((study: any) => {
       if (study[manifestFieldName]) {
-        manifest.push(...study[manifestFieldName]);
+        if ('commons_url' in study) { // HEAL addition to allow hostname based DRS in file access clients
```

Comment:
just to make sure wo want to hard code this and make it a convention for all MDS data rather than make it configurable?

Reply:
I don't want to keep it this way. However, it's faster considering the timeframe of HEAL. Created https://ctds-planx.atlassian.net/browse/PXP-8438 to address this after the HEAL MVP.